src/main/java/com/conveyal/r5/api/util/LegMode.java

```diff
@@ -9,5 +13,12 @@
     //Renting a bicycle
     BICYCLE_RENT,
     //Park & Ride
-    CAR_PARK
+    CAR_PARK;
+
+    /** Return the heaviest/fastest StreetMode for use in stop finding */
+    public static StreetMode legModeSetToDominantStreetMode (Set<LegMode> modes) {
```

Comment:
the expression "set to" sounds like a value is being assigned to a variable ("I set this variable to X"). I don't think the name of the function needs to include the type of its parameter, since that is visible from the signature. I'd just call this "getDominantStreetMode".